### PR TITLE
Adds a single favicon of the little deckard fella

### PR DESCRIPTION
**Pull request by luketlancaster**
Here's a thing that I did that I have no idea if it'll work or not, or if it's the correct way to do things. Address #3

### CODE REVIEW

File: deckard/http/templates/base.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <title>Deckard</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="csrf-token" content="{{ csrf_token() }}">
+    <link rel="icon" type="imgage/ico" href="/static/images/deckard.ico">
```
Comment (paulfalgout) [inline deckard/http/templates/base.html]: type="image/x-icon"

Comment (luketlancaster) [reply deckard/http/templates/base.html]: `imgage`. I think I copied this from my personal site...what a dummy...

Comment (luketlancaster) [reply deckard/http/templates/base.html]: updated @paulfalgout

**Review by paulfalgout — APPROVED** [review]

**Comment by luketlancaster** [comment]
@BenCoffeed so what do we need to do to deploy this out?

**Comment by BenCoffeed** [comment]
🤔 
I either need to figure out how to force-install Deckard w/o using rwell-tasks installing it as a dependency, or we need to bump the Deckard version # in `setup.py` and then modify the Deckard version requirement in rwell-tasks's `setup.py`. Hold plz.

**Pull request merged**